Decred Wallet/Features/Security/PasswordSetupViewController.swift

```diff
@@ -34,32 +50,49 @@ class PasswordSetupViewController: SecurityBaseViewController, UITextFieldDelega
         self.tfPassword.delegate = self
         self.tfConfirmPassword.delegate = self
         
-        self.headerText.text = String(format: LocalizedStrings.createPassword, self.securityFor)
-        self.lbMatchIndicator.text = " " // use empty space so label height isn't reduced
     }
     
     @objc func passwordTextFieldChange() {
+        
         let passwordStrength = PinPasswordStrength.percentageStrength(of: self.tfPassword.text ?? "")
         pbPasswordStrength.progress = passwordStrength.strength
         pbPasswordStrength.progressTintColor = passwordStrength.color
         
+        if self.tfPassword.text == ""{
```

Comment:
Space before opening brace. Also check the else statement and add whitespaces as appropriate.